packages/app/src/systems/Bridge/components/AccountSwitch/AccountSwitch.tsx

```diff
@@ -0,0 +1,81 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Button,
+  Dialog,
+  Stack,
+  Text,
+  Flex,
+  Image,
+  Copyable,
+} from '@fuel-ui/react';
+
+import { formatAddress } from '~/systems/Core/utils';
+
+type AccountSwitchProps = {
+  open: boolean;
+  accounts: string[];
+  onSelect: (val: string) => void;
+  onConnect: () => void;
+  onDisconnect: () => void;
+};
+
+export const AccountSwitch = ({
+  open,
+  accounts,
+  onSelect,
+  onConnect,
+  onDisconnect,
+}: AccountSwitchProps) => {
+  const accountsList = accounts.map((account) => {
+    return (
+      <Flex align="center" gap="$1" key={account}>
+        <Image src="" width="30px" height="30px" />
+        <Text onClick={() => onSelect(account)} color="blackA12">
+          {formatAddress(account)}
+        </Text>
+        <Copyable value={account} />
+      </Flex>
+    );
+  });
+
+  return (
+    <>
+      {open && (
+        <Dialog isOpen={open}>
+          <Dialog.Content css={styles.content}>
```

Comment:
we dont need to control: 
- open prop
- <Dialog>
- <Dialog.Content>

all of that is gonna be handled by  `Overlay` structure introduced by #16

Reply:
component only need to handle: 
<Dialog.Heading>
<Dialog.Description>
<Dialog.Footer>


see: https://github.com/FuelLabs/fuels-wallet/blob/master/packages/app/src/systems/Account/pages/Accounts/Accounts.tsx